uctMaths/uctMaths/background_tasks.py

```diff
@@ -55,8 +55,8 @@ def bg_generate_school_answer_sheets(school_id):
     if not pdf:
         print("%s: Unable to get answer sheets!" % (current_time()))
         return
-    
     send_answer_sheets(school, pdf, True)
+    pdf.close()
```

Comment:
You don't need to close pdf, since it's a `StringIO` and `StringIO` doesn't need to be closed.